games-action/openclonk-7.0

```diff
@@ -0,0 +1,118 @@
+SUMMARY="A free multiplayer action game where you control clonks"
+DESCRIPTION="OpenClonk is a free multiplayer action game in which you\
+control clonks, small, but witty and nimble humanoid beings. The game\
+is mainly about mining, settling and fast-paced melees."
+HOMEPAGE="http://www.openclonk.org/"
+COPYRIGHT="2009-2016 The OpenClonk Team and contributors"
```

Comment:
Missing copyright holder
https://git.openclonk.org/openclonk.git/blob/c54d917f7e85cabebe447259f46ca0092469284e:/COPYING#l1

Reply:
Address to this issue ...

Reply:
Still not complete for the Copyright ...